internal/process/defunct_processes.go

```diff
@@ -0,0 +1,59 @@
+package process
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// DefunctProcesses returns the PIDs of all the zombie processes in the node
+func DefunctProcesses() ([]int, error) {
```

Comment:
I think we should keep the name of the process as well as the pid, so a wrapper struct could help here.
```suggestion
func DefunctProcesses() ([]Process, error) {
```

Reply:
As discussed, we probably only want to implement a simple counter metric, like we do for OOM. This means we do not need the PID and name of the process at all for the metric and can only return the amount here.

It would be still helpful for debugging to have the pid as well as the process name in the logs.